src/commands/Settings/settings.ts

```diff
@@ -40,7 +40,8 @@ module.exports = class SettingsCommand extends Command {
 
     if (!module.success) {
       embed.setDescription('Run `' + settingsData.prefix + 'settings <module>' + '` to see more detailed settings' +
-          '\nModules: ' + listModulesString() +
+          '\nModules: ' + listModulesString() + `\n\nRun ${surroundStringWithBackTick(settingsData.prefix + "togglemodules")} to enable or disable any specific module` + 
```

Comment:
Can you put an extra space between the first two lines and the two lines you added, it feels squished. Also change it so its toggleModules and not togglemodules